src/app/screens/Send.tsx

```diff
@@ -124,7 +124,7 @@ function Send() {
               value={invoice}
               disabled={loading}
               onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
-                setInvoice(event.target.value)
+                setInvoice(event.target.value.trim())
```

Comment:
```suggestion
                setInvoice(event.target.value.trim().replace(/^lightning:/i,""))
```

we should additionally remove a potential `lightning:` in the beginning. This is there if you copy a link.